Increase visibility for SupportedHeaderLines.allHeaderLines

```
[INFO] Compiling 6 Scala sources to target/scala-2.10.6/classes...
[ERROR] .../Freebayes.scala:77: object SupportedHeaderLines in package converters
cannot be accessed in package org.bdgenomics.adam.converters
[ERROR] implicit val uFormatter = new VCFOutFormatter(SupportedHeaderLines.allHeaderLines)
```
In this specific use case I'd like to add Freebayes-specific headers to the ones we already support, something like
```scala
val freebayesHeaderLines = Seq( ... )
val headerLines = SupportedHeaderLines.addHeaderLines ++ freebayesHeaderLines
implicit val uFormatter = new VCFOutFormatter(headerLines)
```

--- a/adam-core/src/main/scala/org/bdgenomics/adam/converters/DefaultHeaderLines.scala
+++ b/adam-core/src/main/scala/org/bdgenomics/adam/converters/DefaultHeaderLines.scala
@@ -27,9 +27,9 @@ import htsjdk.variant.vcf.{
 }
 
 /**
- * All header lines for VCF INFO and GT format fields that are supported in ADAM.
+ * Default set of VCF header lines for INFO and GT format fields supported in ADAM.
  */
-private[adam] object SupportedHeaderLines {
+object DefaultHeaderLines {
 
   lazy val ancestralAllele = new VCFInfoHeaderLine("AA",
     1,
@@ -89,7 +89,7 @@ private[adam] object SupportedHeaderLines {
     "Functional annotations: 'Allele | Annotation | Annotation_Impact | Gene_Name | Gene_ID | Feature_Type | Feature_ID | Transcript_BioType | Rank | HGVS.c | HGVS.p | cDNA.pos / cDNA.length | CDS.pos / CDS.length | AA.pos / AA.length | Distance | ERRORS / WARNINGS / INFO'")
 
   /**
-   * All info keys in VCF format.
+   * Default set of VCF header lines for INFO fields supported in ADAM.
    */
   lazy val infoHeaderLines = Seq(
     ancestralAllele,
@@ -167,7 +167,7 @@ private[adam] object SupportedHeaderLines {
     "Total number of reads with mapping quality=0")
 
   /**
-   * All format lines in VCF format.
+   * Default set of VCF header lines for GT format fields supported in ADAM.
    */
   lazy val formatHeaderLines = Seq(
     genotype,
@@ -186,7 +186,7 @@ private[adam] object SupportedHeaderLines {
   )
 
   /**
-   * All supported header lines in VCF format.
+   * Default set of VCF header lines for INFO and GT format fields supported in ADAM.
    */
   lazy val allHeaderLines = infoHeaderLines ++ formatHeaderLines
 }

--- a/adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantContextConverter.scala
+++ b/adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantContextConverter.scala
@@ -1394,7 +1394,7 @@ private[adam] class VariantContextConverter(
           val key = il.getID
 
           // filter out the lines that we already support
-          if (SupportedHeaderLines.infoHeaderLines
+          if (DefaultHeaderLines.infoHeaderLines
             .find(_.getID == key)
             .isEmpty) {
 
@@ -1467,7 +1467,7 @@ private[adam] class VariantContextConverter(
           val key = fl.getID
 
           // filter out the lines that we already support
-          if (SupportedHeaderLines.formatHeaderLines
+          if (DefaultHeaderLines.formatHeaderLines
             .find(_.getID == key)
             .isEmpty) {
 
@@ -1818,7 +1818,7 @@ private[adam] class VariantContextConverter(
           val key = il.getID
 
           // filter out the lines that we already support
-          if (SupportedHeaderLines.infoHeaderLines
+          if (DefaultHeaderLines.infoHeaderLines
             .find(_.getID == key)
             .isDefined) {
 
@@ -1891,7 +1891,7 @@ private[adam] class VariantContextConverter(
           val key = fl.getID
 
           // filter out the lines that we already support
-          if (SupportedHeaderLines.formatHeaderLines
+          if (DefaultHeaderLines.formatHeaderLines
             .find(_.getID == key)
             .isDefined) {
 

--- a/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala
+++ b/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala
@@ -239,7 +239,7 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
     deduped.flatMap(line => line match {
       case fl: VCFFormatHeaderLine => {
         val key = fl.getID
-        SupportedHeaderLines.formatHeaderLines
+        DefaultHeaderLines.formatHeaderLines
           .find(_.getID == key)
           .fold(Some(fl).asInstanceOf[Option[VCFCompoundHeaderLine]])(defaultLine => {
             auditLine(fl, defaultLine, (newId, oldLine) => {
@@ -252,7 +252,7 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
       }
       case il: VCFInfoHeaderLine => {
         val key = il.getID
-        SupportedHeaderLines.infoHeaderLines
+        DefaultHeaderLines.infoHeaderLines
           .find(_.getID == key)
           .fold(Some(il).asInstanceOf[Option[VCFCompoundHeaderLine]])(defaultLine => {
             auditLine(il, defaultLine, (newId, oldLine) => {
@@ -266,7 +266,7 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
       case l => {
         Some(l)
       }
-    }) ++ SupportedHeaderLines.allHeaderLines
+    }) ++ DefaultHeaderLines.allHeaderLines
   }
 
   private def headerLines(header: VCFHeader): Seq[VCFHeaderLine] = {

--- a/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/ADAMVCFOutputFormat.scala
+++ b/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/ADAMVCFOutputFormat.scala
@@ -20,7 +20,6 @@ package org.bdgenomics.adam.rdd.variant
 import htsjdk.variant.vcf.{ VCFHeaderLine, VCFHeader }
 import org.apache.hadoop.fs.{ FileSystem, Path }
 import org.apache.hadoop.mapreduce.{ RecordWriter, TaskAttemptContext }
-import org.bdgenomics.adam.converters.SupportedHeaderLines
 import org.bdgenomics.adam.models.SequenceDictionary
 import org.seqdoop.hadoop_bam.{
   KeyIgnoringVCFOutputFormat,

--- a/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala
+++ b/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala
@@ -20,7 +20,7 @@ package org.bdgenomics.adam.rdd.variant
 import htsjdk.samtools.ValidationStringency
 import htsjdk.variant.vcf.VCFHeaderLine
 import org.apache.spark.rdd.RDD
-import org.bdgenomics.adam.converters.SupportedHeaderLines
+import org.bdgenomics.adam.converters.DefaultHeaderLines
 import org.bdgenomics.adam.models.{
   ReferencePosition,
   ReferenceRegion,
@@ -73,7 +73,7 @@ private[adam] class GenotypeArraySerializer extends IntervalArraySerializer[Refe
 case class GenotypeRDD(rdd: RDD[Genotype],
                        sequences: SequenceDictionary,
                        @transient samples: Seq[Sample],
-                       @transient headerLines: Seq[VCFHeaderLine] = SupportedHeaderLines.allHeaderLines) extends MultisampleAvroGenomicRDD[Genotype, GenotypeRDD] {
+                       @transient headerLines: Seq[VCFHeaderLine] = DefaultHeaderLines.allHeaderLines) extends MultisampleAvroGenomicRDD[Genotype, GenotypeRDD] {
 
   protected def buildTree(rdd: RDD[(ReferenceRegion, Genotype)])(
     implicit tTag: ClassTag[Genotype]): IntervalArray[ReferenceRegion, Genotype] = {

--- a/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantContextRDD.scala
+++ b/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantContextRDD.scala
@@ -24,7 +24,7 @@ import org.apache.hadoop.io.LongWritable
 import org.apache.hadoop.fs.Path
 import org.apache.spark.rdd.RDD
 import org.bdgenomics.adam.converters.{
-  SupportedHeaderLines,
+  DefaultHeaderLines,
   VariantContextConverter
 }
 import org.bdgenomics.adam.models.{
@@ -86,7 +86,7 @@ private[adam] class VariantContextArraySerializer extends IntervalArraySerialize
 case class VariantContextRDD(rdd: RDD[VariantContext],
                              sequences: SequenceDictionary,
                              @transient samples: Seq[Sample],
-                             @transient headerLines: Seq[VCFHeaderLine] = SupportedHeaderLines.allHeaderLines) extends MultisampleGenomicRDD[VariantContext, VariantContextRDD]
+                             @transient headerLines: Seq[VCFHeaderLine] = DefaultHeaderLines.allHeaderLines) extends MultisampleGenomicRDD[VariantContext, VariantContextRDD]
     with Logging {
 
   protected def buildTree(rdd: RDD[(ReferenceRegion, VariantContext)])(

--- a/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantRDD.scala
+++ b/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantRDD.scala
@@ -21,7 +21,7 @@ import htsjdk.samtools.ValidationStringency
 import htsjdk.variant.vcf.{ VCFHeader, VCFHeaderLine }
 import org.apache.hadoop.fs.Path
 import org.apache.spark.rdd.RDD
-import org.bdgenomics.adam.converters.SupportedHeaderLines
+import org.bdgenomics.adam.converters.DefaultHeaderLines
 import org.bdgenomics.adam.models.{
   ReferenceRegion,
   ReferenceRegionSerializer,
@@ -78,7 +78,7 @@ private[adam] class VariantArraySerializer extends IntervalArraySerializer[Refer
  */
 case class VariantRDD(rdd: RDD[Variant],
                       sequences: SequenceDictionary,
-                      @transient headerLines: Seq[VCFHeaderLine] = SupportedHeaderLines.allHeaderLines) extends AvroGenomicRDD[Variant, VariantRDD] {
+                      @transient headerLines: Seq[VCFHeaderLine] = DefaultHeaderLines.allHeaderLines) extends AvroGenomicRDD[Variant, VariantRDD] {
 
   protected def buildTree(rdd: RDD[(ReferenceRegion, Variant)])(
     implicit tTag: ClassTag[Variant]): IntervalArray[ReferenceRegion, Variant] = {

--- a/adam-core/src/test/scala/org/bdgenomics/adam/converters/VariantContextConverterSuite.scala
+++ b/adam-core/src/test/scala/org/bdgenomics/adam/converters/VariantContextConverterSuite.scala
@@ -53,9 +53,9 @@ class VariantContextConverterSuite extends ADAMFunSuite {
   }
 
   val lenient = ValidationStringency.LENIENT
-  val converter = new VariantContextConverter(SupportedHeaderLines.allHeaderLines,
+  val converter = new VariantContextConverter(DefaultHeaderLines.allHeaderLines,
     lenient)
-  val strictConverter = new VariantContextConverter(SupportedHeaderLines.allHeaderLines,
+  val strictConverter = new VariantContextConverter(DefaultHeaderLines.allHeaderLines,
     ValidationStringency.STRICT)
 
   def htsjdkSNVBuilder: VariantContextBuilder = new VariantContextBuilder()
@@ -1883,7 +1883,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Flag,
       "Flag")
 
-    val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ flagHeader, lenient)
+    val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ flagHeader, lenient)
       .convert(adamVc).orNull
 
     assert(vc.hasAttribute("FLAG"))
@@ -1917,7 +1917,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
      * this behavior.
      */
     intercept[IllegalArgumentException] {
-      new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ flagHeader,
+      new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ flagHeader,
         ValidationStringency.STRICT)
     }
   }
@@ -1934,7 +1934,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=1 Type=Integer")
 
-    val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ oneIntHeader, lenient)
+    val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ oneIntHeader, lenient)
       .convert(adamVc).orNull
 
     assert(vc.hasAttribute("ONE_INT"))
@@ -1953,7 +1953,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=4 Type=Integer")
 
-    val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ fourIntsHeader, lenient)
+    val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ fourIntsHeader, lenient)
       .convert(adamVc).orNull
 
     assert(vc.hasAttribute("FOUR_INTS"))
@@ -1976,7 +1976,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=A Type=Integer")
 
-    val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ aIntHeader, lenient)
+    val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ aIntHeader, lenient)
       .convert(adamVc).orNull
 
     assert(vc.hasAttribute("A_INT"))
@@ -1995,7 +1995,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=R Type=Integer")
 
-    val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ rIntHeader, lenient)
+    val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ rIntHeader, lenient)
       .convert(adamVc).orNull
 
     assert(vc.hasAttribute("R_INT"))
@@ -2016,7 +2016,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.String,
       "Number=R Type=String")
 
-    val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ rStringHeader, lenient)
+    val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ rStringHeader, lenient)
       .convert(adamVc).orNull
 
     assert(vc.hasAttribute("R_STRING"))
@@ -2038,7 +2038,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       "Number=G Type=String")
 
     intercept[IllegalArgumentException] {
-      val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ gStringHeader, lenient)
+      val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ gStringHeader, lenient)
         .convert(adamVc).orNull
     }
   }
@@ -2053,7 +2053,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Flag,
       "Flag")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ flagHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ flagHeader, lenient)
       .convert(vc)
       .head
 
@@ -2072,7 +2072,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=1 Type=Integer")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ oneIntHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ oneIntHeader, lenient)
       .convert(vc)
       .head
 
@@ -2092,7 +2092,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=4 Type=Integer")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ fourIntsHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ fourIntsHeader, lenient)
       .convert(vc)
       .head
 
@@ -2112,7 +2112,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Float,
       "Number=4 Type=Float")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ fourFloatsHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ fourFloatsHeader, lenient)
       .convert(vc)
       .head
 
@@ -2131,7 +2131,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=A Type=Integer")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ aIntHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ aIntHeader, lenient)
       .convert(vc)
       .head
 
@@ -2150,7 +2150,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=R Type=Integer")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ rIntHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ rIntHeader, lenient)
       .convert(vc)
       .head
 
@@ -2169,7 +2169,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.String,
       "Number=R Type=String")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ rStringHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ rStringHeader, lenient)
       .convert(vc)
       .head
 
@@ -2189,7 +2189,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       "Number=G Type=String")
 
     intercept[IllegalArgumentException] {
-      val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ gStringHeader, lenient)
+      val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ gStringHeader, lenient)
         .convert(vc)
     }
   }
@@ -2210,7 +2210,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
         VCFHeaderLineType.Flag,
         "Flag")
 
-      val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ flagHeader, lenient)
+      val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ flagHeader, lenient)
         .convert(adamVc).orNull
     }
   }
@@ -2231,7 +2231,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=1 Type=Integer")
 
-    val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ oneIntHeader, lenient)
+    val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ oneIntHeader, lenient)
       .convert(adamVc).orNull
 
     assert(vc.hasGenotypes)
@@ -2256,7 +2256,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=4 Type=Integer")
 
-    val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ fourIntsHeader, lenient)
+    val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ fourIntsHeader, lenient)
       .convert(adamVc).orNull
 
     assert(vc.hasGenotypes)
@@ -2286,7 +2286,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=A Type=Integer")
 
-    val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ aIntHeader, lenient)
+    val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ aIntHeader, lenient)
       .convert(adamVc).orNull
 
     assert(vc.hasGenotypes)
@@ -2313,7 +2313,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=R Type=Integer")
 
-    val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ rIntHeader, lenient)
+    val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ rIntHeader, lenient)
       .convert(adamVc).orNull
 
     assert(vc.hasGenotypes)
@@ -2341,7 +2341,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.String,
       "Number=R Type=String")
 
-    val vc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ rStringHeader, lenient)
+    val vc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ rStringHeader, lenient)
       .convert(adamVc).orNull
 
     assert(vc.hasGenotypes)
@@ -2364,7 +2364,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
         VCFHeaderLineType.Flag,
         "Flag")
 
-      val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ flagHeader, lenient)
+      val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ flagHeader, lenient)
         .convert(vc)
         .head
 
@@ -2387,7 +2387,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=1 Type=Integer")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ oneIntHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ oneIntHeader, lenient)
       .convert(vc)
       .head
 
@@ -2410,7 +2410,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=4 Type=Integer")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ fourIntsHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ fourIntsHeader, lenient)
       .convert(vc)
       .head
 
@@ -2433,7 +2433,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Float,
       "Number=4 Type=Float")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ fourFloatsHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ fourFloatsHeader, lenient)
       .convert(vc)
       .head
 
@@ -2456,7 +2456,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=A Type=Integer")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ aIntHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ aIntHeader, lenient)
       .convert(vc)
       .head
 
@@ -2479,7 +2479,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.Integer,
       "Number=R Type=Integer")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ rIntHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ rIntHeader, lenient)
       .convert(vc)
       .head
 
@@ -2502,7 +2502,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.String,
       "Number=R Type=String")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ rStringHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ rStringHeader, lenient)
       .convert(vc)
       .head
 
@@ -2525,7 +2525,7 @@ class VariantContextConverterSuite extends ADAMFunSuite {
       VCFHeaderLineType.String,
       "Number=G Type=String")
 
-    val adamVc = new VariantContextConverter(SupportedHeaderLines.allHeaderLines :+ gStringHeader, lenient)
+    val adamVc = new VariantContextConverter(DefaultHeaderLines.allHeaderLines :+ gStringHeader, lenient)
       .convert(vc)
       .head
 

--- a/adam-core/src/test/scala/org/bdgenomics/adam/models/ReferenceRegionSuite.scala
+++ b/adam-core/src/test/scala/org/bdgenomics/adam/models/ReferenceRegionSuite.scala
@@ -24,7 +24,7 @@ import htsjdk.variant.variantcontext.{
   VariantContextBuilder
 }
 import org.bdgenomics.adam.converters.{
-  SupportedHeaderLines,
+  DefaultHeaderLines,
   VariantContextConverter
 }
 import org.bdgenomics.formats.avro._
@@ -444,7 +444,7 @@ class ReferenceRegionSuite extends FunSuite {
   }
 
   test("convert a genotype and then get the reference region") {
-    val converter = new VariantContextConverter(SupportedHeaderLines.allHeaderLines,
+    val converter = new VariantContextConverter(DefaultHeaderLines.allHeaderLines,
       ValidationStringency.LENIENT)
     val vcb = new VariantContextBuilder()
       .alleles(List(Allele.create("A", true), Allele.create("T")))

--- a/adam-core/src/test/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDSuite.scala
+++ b/adam-core/src/test/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDSuite.scala
@@ -20,7 +20,7 @@ package org.bdgenomics.adam.rdd.read
 import java.io.File
 import java.nio.file.Files
 import htsjdk.samtools.ValidationStringency
-import org.bdgenomics.adam.converters.SupportedHeaderLines
+import org.bdgenomics.adam.converters.DefaultHeaderLines
 import org.bdgenomics.adam.models.{
   RecordGroupDictionary,
   ReferenceRegion,
@@ -699,7 +699,7 @@ class AlignmentRecordRDDSuite extends ADAMFunSuite {
     val ardd = sc.loadBam(readsPath)
 
     implicit val tFormatter = SAMInFormatter
-    implicit val uFormatter = new VCFOutFormatter(SupportedHeaderLines.allHeaderLines)
+    implicit val uFormatter = new VCFOutFormatter(DefaultHeaderLines.allHeaderLines)
 
     val pipedRdd: VariantContextRDD = ardd.pipe("/bin/bash $0 %s $1".format(tempPath),
       files = Seq(scriptPath, vcfPath))

